### PR TITLE
Add Logic files And Handling Event

### PR DESCRIPTION
**Pull request by Cosmosha**
The following requirement was implemented;

- Add Logic directory 
- Add operate and calculate files
-  Implementing Click event handlers
- Using props and state

### CODE REVIEW

File: src/components/calculationButtons.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <button type="button" onClick={() => calcAction('5')}> 5 </button>
+      <button type="button" onClick={() => calcAction('6')}> 6 </button>
+      <button type="button" className="operators" onClick={() => calcAction('-')}> - </button>
+      <button type="button" onClick={() => calcAction('1')}> 1 </button>
+      <button type="button" onClick={() => calcAction('2')}> 2 </button>
+      <button type="button" onClick={() => calcAction('3')}> 3 </button>
+      <button type="button" className="operators" onClick={() => calcAction('+')}> + </button>
+      <button type="button" className="span-two" onClick={() => calcAction('0')}> 0 </button>
+      <button type="button" onClick={() => calcAction('.')}> . </button>
+      <button type="button" className="operators" onClick={() => calcAction('=')}> = </button>
```
Comment (OLIPLICHE) [inline src/components/calculationButtons.js]: - Hi, After checking on your App, I noticed that the `division` button does not exist, also the `%` button is not working as expected. I would advise you to fix these buttons and make them work as expected. This project can land you a better opportunity, it is not good to submit an app that works with mistakes.
-

Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Cosmosha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The calculate App handles state with useState
- Linter is green (the last commit has a green check mark (✅)) 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculationButtons.js (hunk 1 above)
Comment (Cosmosha) [reply src/components/calculationButtons.js]: Hi, 
thank you for the feedback, I will do well to add the division('/') sign. However, the modulo ('%') is working as expected so please recheck. Thank you.

**Review by sumairq — APPROVED** [review]
### Hi @Cosmosha , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**